src/forms/form-control.scss

```
@@ -58,15 +74,7 @@ label {
   }
 
   &[disabled] {
```

I would have to test to be sure, but we might can extend this selector to include the `fieldset` like:

```suggestion
  &[disabled],
  fieldset[disabled] & {
```

I think that should compile to:

```
.form-control[disabled],
.form-select[disabled],
fieldset[disabled] .form-control,
fieldset[disabled] .form-select {
```

then we don't need the mixin and wouldn't output the same styles twice, saving a tiny bit of file size.

---

Nice, I like this a lot better. I'll make the proposed changes and test.